package.json

```
@@ -1,12 +1,14 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-TAT",
+  "name": "engagesphere-test-design-masterclass-tat",
   "version": "1.0.0",
   "description": "Sample project with basic 'backend' and frontend written in JavaScript.",
   "scripts": {
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "test-headed": "cypress open",
+    "test-headless": "cypress run --headless"
```

```suggestion
    "test-headless": "cypress run"
```

O padrão do `cypress run` já é _headless_, portanto, isso se torna redundante.